R/verbosity.R

```diff
@@ -74,9 +74,8 @@ with_lifecycle_silence <- function(expr) {
 #' @export
 scoped_lifecycle_warnings <- function(frame = caller_env()) {
   scoped_options(.frame = frame,
-    lifecycle_disable_warnings = FALSE,
-    lifecycle_verbose_soft_deprecation = TRUE,
-    lifecycle_repeat_warnings = TRUE
+    lifecycle_shush_warnings = FALSE,
```

Comment:
shush -> quiet?

Reply:
same number of letters as "force" so all good :-)